src/Orchard.Specs/Boolean.feature

```diff
@@ -48,7 +48,7 @@ Scenario: Creating and using Boolean fields
     When I go to "Admin/ContentTypes/Edit/Event"
         And I fill in 
             | name                                | value                        |
-            | Fields[0].BooleanFieldSettings.Hint | Check if the event is active |
```

Comment:
This change is weird ... did you find which commit did it?

Reply:
I did a PR a while back where I changed the names written in the generated html for fields and such, because they would not have the same index when opened in two browser tabs at the same time.

The repro for the issue was to open a content definition on two tabs, update it from one, then try to update it from the other. After saving on the first tab, the order of fields in memory would be different than the one in the html of the second tab, hence saving again there would potentially mess settings up.

I can dig the issue and PR up for more details if you like.